lib/base/logger.hpp

```diff
@@ -6,7 +6,7 @@
 #include "base/i2-base.hpp"
 #include "base/logger-ti.hpp"
 #include <set>
-#include <iosfwd>
+#include <sstream>
```

Comment:
I don't get this this change. There's no change regarding any iostreams, so why is this needed?

Reply:
https://github.com/Icinga/icinga2/runs/1813342849?check_suite_focus=true

Reply:
Ok, and this was probably included by some of the two boost headers which was basically included everywhere. Fine.